cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/profiles_populator.py

```diff
@@ -252,75 +244,68 @@ def validate_profile(profile: AzureProfile) -> bool:
     is_valid = True
     cli_tell(f"Profile name: {profile.name}")
 
-    # check invariants
     try:
-        raise_on_missing_fields(profile)
-        raise_on_invalid_storage(profile)
-    except (ProfilesInvalidError, ProfilesIncompleteError) as err:
-        cli_tell(str(err))
-        is_valid = False
+        # check invariants
+        validate_profile_configuration(profile)
```

Comment:
`validate_profile_configuration` can be method of `AzureProfile`

Reply:
It can
:)